### ISSUE TITLE
aircraft catalogue

### ISSUE THREAD

**Issue by monkeypants** [monkeypants/UASFunPolice#5]
(stub ticket)

we need to know what aircraft exist before we can:
- manage their configuration
- link them to incidents,
- manage their flight logs, etc.

Operationally, we might want a little per-aircraft configuration that links the aircraft name (and log directory) to the catalogue identity. That way, we could automate management of telemetry data for that aircraft.

It might be nice to have a "current fleet" and "wall of fame" on the web site.
